### ISSUE TITLE
[BNNS] Chromium crashed and then reloaded when running cts tests with BNNS backend

### ISSUE THREAD

**Issue by Christywl** [intel/webml-polyfill#147]
**Test Env:**
Chromium Version : nightly build 70.0.3503.0 (**public/webml/nightly/007f215/**)
OS : Mac OS
webml-polyfill: commit 777b85f

**Expected Result:**
No crash.

**Actual Result:**
Chromium crashed and then reloaded when running cts tests.

**How to Reproduce:**
1. Launch chromium and enable the WebML
2. Run the tests: https://brucedai.github.io/nt/test/cts-all.html?backend=bnns 

Crash log:
```
[11060:775:0816/151803.529907:ERROR:native_view_host_mac.mm(177)] Not implemented reached in virtual void views::NativeViewHostMac::ShowWidget(int, int, int, int, int, int)
[11060:775:0816/151803.529942:ERROR:native_view_host_mac.mm(177)] Not implemented reached in virtual void views::NativeViewHostMac::ShowWidget(int, int, int, int, int, int)
[11076:775:0816/151810.927418:ERROR:webthread_impl_for_utility_thread.cc(19)] Not implemented reached in virtual blink::ThreadScheduler *blink::scheduler::WebThreadImplForUtilityThread::Scheduler() const
[11060:775:0816/151811.267755:ERROR:events_mac.mm(66)] Not implemented reached in ui::EventType ui::EventTypeFromNative(const ui::PlatformEvent &)17
[11060:775:0816/151811.311232:ERROR:events_mac.mm(66)] Not implemented reached in ui::EventType ui::EventTypeFromNative(const ui::PlatformEvent &)17
[11065:775:0816/151815.397222:ERROR:compilation_impl_mac.mm(667)] ********FUSE_CODE POOL*********0
[11065:775:0816/151815.488008:ERROR:compilation_impl_mac.mm(667)] ********FUSE_CODE POOL*********0
[11065:775:0816/151815.504410:ERROR:compilation_impl_mac.mm(667)] ********FUSE_CODE POOL*********0
[11065:775:0816/151815.583631:ERROR:compilation_impl_mac.mm(667)] ********FUSE_CODE POOL*********3
[11065:775:0816/151815.602974:ERROR:compilation_impl_mac.mm(951)] CompilationImplMac::CompileConcatenation
[11065:775:0816/151815.607832:ERROR:compilation_impl_mac.mm(951)] CompilationImplMac::CompileConcatenation
[11065:775:0816/151815.618835:ERROR:compilation_impl_mac.mm(951)] CompilationImplMac::CompileConcatenation
Received signal 11 <unknown> 000000000000
 [0x00011c3d86ec]
 [0x00011c3d85e1]
 [0x7fff5f08df5a]
 [0x00010a82a080]
 [0x00011b11b01a]
 [0x00011b11b39e]
 [0x00011b119d9c]
 [0x00011c9c27ac]
 [0x00011c9c6656]
 [0x00011c9c4b02]
 [0x00011c9c38ed]
 [0x00011c9bf96b]
 [0x00011c9004b8]
 [0x00011c327f32]
 [0x00011c348adf]
 [0x00011c348fa8]
 [0x00011c34b16a]
 [0x00011c33b1ea]
 [0x00011c34aa8f]
 [0x7fff36a83a61]
 [0x7fff36b3d47c]
 [0x7fff36a664c0]
 [0x7fff36a6593d]
 [0x7fff36a651a3]
 [0x7fff38b54f26]
 [0x00011c34b7cd]
 [0x00011c34a5ae]
 [0x00011c36d555]
 [0x00012044a34c]
 [0x00011bf1f729]
 [0x00011d792a01]
 [0x00011bf1e8f4]
 [0x00011a010a03]
 [0x00010a807131]
 [0x7fff5ed7f015]
 [0x000000000007]
[end of stack trace]
[11078:775:0816/151816.322037:ERROR:gles2_cmd_decoder.cc(18059)] [.BrowserCompositor-0x7fa7a8034800]GL ERROR :GL_INVALID_OPERATION : glCreateAndConsumeTextureCHROMIUM: invalid mailbox name
[11078:775:0816/151816.322213:ERROR:gles2_cmd_decoder.cc(12532)] [.BrowserCompositor-0x7fa7a8034800]GL ERROR :GL_INVALID_VALUE : glScheduleCALayerCHROMIUM: unsupported texture format
```

**Comment by Christywl** [comment]
This issue didn't happen on chromium nightly build **e7980af**.

**Comment by Christywl** [comment]
https://github.com/intel/webml-polyfill/issues/149#issuecomment-423256847
> Some test cases crashed and broke the neural network service

The crash issue still happened for BNNS backend on the nightly build 861b761, and 'Neural Network service unavailable' occurred.
Test URL: https://brucedai.github.io/nt/testm/index-local.html?backend=bnns
<img width="1280" alt="screen shot 2018-10-08 at 10 59 08 am" src="https://user-images.githubusercontent.com/20039821/46591119-1b5cec00-caeb-11e8-9fd8-204040080659.png">

**Comment by lisa0314** [comment]
Received signal 11 <unknown> 000000000000
0   Chromium Framework                  0x000000011c33dafe base::debug::StackTrace::StackTrace(unsigned long) + 174
1   Chromium Framework                  0x000000011c33db5d base::debug::StackTrace::StackTrace(unsigned long) + 29
2   Chromium Framework                  0x000000011bf989dc base::debug::StackTrace::StackTrace() + 28
3   Chromium Framework                  0x000000011c33d8ef base::debug::(anonymous namespace)::StackDumpSignalHandler(int, __siginfo*, void*) + 1407
4   libsystem_platform.dylib            0x00007fff767fcf5a _sigtramp + 26
5   Chromium Framework                  0x000000012f80dc0b angle::(anonymous namespace)::g_angle_reset_platform + 252563
6   Chromium Framework                  0x00000001172b62c4 ml::ExecutionImplMac::~ExecutionImplMac() + 1764
7   Chromium Framework                  0x00000001172b6435 ml::ExecutionImplMac::~ExecutionImplMac() + 21
8   Chromium Framework                  0x00000001172b6459 ml::ExecutionImplMac::~ExecutionImplMac() + 25
9   Chromium Framework                  0x00000001172b37bc mojo::StrongBinding<ml::mojom::Execution>::~StrongBinding() + 252
10  Chromium Framework                  0x00000001172b36b5 mojo::StrongBinding<ml::mojom::Execution>::~StrongBinding() + 21
11  Chromium Framework                  0x00000001172b3687 mojo::StrongBinding<ml::mojom::Execution>::Close() + 39
12  Chromium Framework                  0x00000001172b1cb5 mojo::StrongBinding<ml::mojom::Execution>::OnConnectionError(unsigned int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&) + 149
13  Chromium Framework                  0x00000001172b325f void base::internal::FunctorTraits<void (mojo::StrongBinding<ml::mojom::Execution>::*)(unsigned int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&), void>::Invoke<void (mojo::StrongBinding<ml::mojom::Execution>::*)(unsigned int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&), mojo::StrongBinding<ml::mojom::Execution>*, unsigned int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&>(void (mojo::StrongBinding<ml::mojom::Execution>::*)(unsigned int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&), mojo::StrongBinding<ml::mojom::Execution>*&&, unsigned int&&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&&&) + 159
@fujunwei , These are some of  the error stack infos.

**Comment by Christywl** [comment]
This issue was fixed on nightly build 16d3dad.
